packages/xstate-graph/package.json

```diff
@@ -14,19 +14,23 @@
   "homepage": "https://github.com/davidkpiano/xstate/tree/master/packages/xstate-test#readme",
   "license": "MIT",
   "main": "lib/index.js",
+  "module": "es/index.js",
```

Comment:
@davidkpiano what's the status of `@xstate/graph`? it has not been included in [`publishablePackages`](https://github.com/davidkpiano/xstate/blob/6bda76dec5b5e08b7da432a2b3c35ef9cc44f8e7/.github/actions/change-release-intent/main.js#L26), but i don't recall why. 

When I look at this now it seems that we should just add it to that set of `publishablePackages` because it's a dependency of `@xstate/test` and is already published and on the v1.

Reply:
It should be - it is published.